centreon/src/App/MonitoringConfiguration/Domain/Aggregate/Command/CommandTypeEnum.php

```diff
@@ -29,4 +29,15 @@ enum CommandTypeEnum: int
     case Check = 2;
     case Miscellaneous = 3;
     case Discovery = 4;
+
+    public static function fromName(string $name): ?self
+    {
+        return match($name) {
+            'Notification' => self::Notification,
+            'Check' => self::Check,
+            'Miscellaneous' => self::Miscellaneous,
+            'Discovery' => self::Discovery,
```

Comment:
```suggestion
            self::Notification->name => self::Notification,
            self::Check->name => self::Check,
            self::Miscellaneous->name => self::Miscellaneous,
            self::Discovery->name => self::Discovery,
```
To avoid magic string and mistakes if we change the case definitions

Reply:
done